virttest/env_process.py

```
@@ -1394,31 +1396,45 @@ def postprocess(test, params, env):
     else:
         screendump_temp_dir = test.debugdir
-    ppm_file_rex = "*_iter%s.ppm" % test.iteration
-    for f in glob.glob(os.path.join(screendump_temp_dir, ppm_file_rex)):
-        if not ppm_utils.image_verify_ppm_file(f):
-            LOG.warning("Found corrupt PPM file: %s", f)
+
+    image_patterns = [
+        ("*_iter%s.ppm" % test.iteration, ppm_utils.image_verify_ppm_file, "PPM"),
+        ("*_iter%s.png" % test.iteration, png_utils.image_verify_png_file, "PNG"),
+    ]
+    for pattern, verify_func, image_format in image_patterns:
```

Ruff (0.14.8)

Loop control variable `image_format` not used within loop body

(B007)